cli/integration_tests/basic_monorepo/monorepo/packages/util/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "util",
+  "scripts": {
+    "build": "echo 'building'"
```

Comment:
might want to create a file `foo`, since the turbo.json is referring to that in `outputs`